source/utilities/server.ts

```diff
@@ -65,7 +65,8 @@ export const startServer = async (
   const sslCert = args['--ssl-cert'];
   const sslKey = args['--ssl-key'];
   const sslPass = args['--ssl-pass'];
-  const isPFXFormat = sslCert && /[.](?<extension>pfx|p12)$/.exec(sslCert);
+  const isPFXFormat =
+    sslCert && /[.](?<extension>pfx|p12)$/.exec(sslCert) !== null;
```

Comment:
`undefined` is not going to work for this as regex.exec returns a array in case of a match and null when not matched. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#return_value)

I have verified that a undefined check does not satisfy the conditions.

```js
/[.](?<extension>pfx|p12)$/.exec('a.pfx') !== null;
// Returns true. Expected. true

/[.](?<extension>pfx|p12)$/.exec('apfx') !== null;
// Returns false. Expected false

/[.](?<extension>pfx|p12)$/.exec('a.pfx') !== undefined;
// Returns true. Expected true

/[.](?<extension>pfx|p12)$/.exec('apfx') !== undefined;
// Returns true. Expected false

```

Reply:
Ohk, I didn't know that! We can keep it null then.